src/moveit_python/move_group_interface.py

```diff
@@ -42,10 +42,10 @@ class MoveGroupInterface(object):
     ## @param frame Name of the fixed frame in which planning happens
     ## @param listener A TF listener instance (optional, will create a new one if None)
     ## @param plan_only Should we only plan, but not execute?
-    def __init__(self, group, frame, listener=None, plan_only=False):
+    def __init__(self, group, frame, move_group="move_group", listener=None, plan_only=False):
```

Comment:
to avoid potentially breaking code, please put the new parameter at the end of the parameter list.

Reply:
@mikeferguson I added move_group explanation to parameter list.
Please check it!